nixos/modules/services/networking/nebula.nix

```diff
@@ -200,8 +229,8 @@ in
               Restart = "always";
               ExecStart = "${netCfg.package}/bin/nebula -config ${configFile}";
               UMask = "0027";
-              CapabilityBoundingSet = "CAP_NET_ADMIN";
-              AmbientCapabilities = "CAP_NET_ADMIN";
+              CapabilityBoundingSet = if settings.lighthouse.serve_dns then "CAP_NET_ADMIN CAP_NET_BIND_SERVICE" else "CAP_NET_ADMIN";
+              AmbientCapabilities = if settings.lighthouse.serve_dns then "CAP_NET_ADMIN CAP_NET_BIND_SERVICE" else "CAP_NET_ADMIN";
```

Comment:
Factor these out into a function. Also, something like this would be useful:

`if (settings.lighthouse.serve_dns && settings.lighthouse.dns.port <= 1024) ...`

So that way we don't give CAP_NET_BIND_SERVICE if it's not necessary.